### PR TITLE
open-cluster-management/backlog#2504 gettign started changes and chan…

### PR DESCRIPTION
**Pull request by swopebe**
…ges to arch topic.

### CODE REVIEW

File: about/architecture.adoc [hunk 1]
```diff
@@ -53,5 +52,6 @@ Governance and risk is the term used to define the processes that are used to ma
 After you configure a Red Hat Advanced Cluster Management for Kubernetes hub cluster and a managed cluster, you can view and create policies with the Red Hat Advanced Cluster Management policy framework.
 For more information about Governance and risk, as well as role-based access control, see the link:../security[Security] introduction.
 
-See the product link:../install[Installing] section to prepare your cluster and get configuration information.
-See the link:../about[Components] topic to learn more about what is installed with the product.
+See the product link:../install/install_overview.adoc[Installing] section to prepare your cluster and get configuration information.
```
Comment (swopebe) [inline about/architecture.adoc:55]: broken links

File: about/quick_start.adoc [hunk 2]
```diff
@@ -4,14 +4,16 @@
 [#introduction]
 == Introduction
 
-Review the xref:../about/components.adoc#components[Components] topics to learn more about what is installed with Red Hat Advanced Cluster Management for Kubernetes.
+See the product architecture at xref:../about/architecture.adoc[Architecture]. Review the xref:../about/components.adoc#components[Components] topics to learn more about what is installed with Red Hat Advanced Cluster Management for Kubernetes when you accept the license and subscription agreement.
```
Comment (swopebe) [inline about/quick_start.adoc:8]: links and from marketing call, I am adding lines in a couple of spots about agreements at their request. Line 7 may change.

File: about/quick_start.adoc [hunk 3]
```diff
@@ -4,14 +4,16 @@
...
-After you learn about the hub cluster and managed cluster architecture, learn more about the link:../manage_cluster#supported-clouds[Supported clouds], which lists the cloud provider cluster options.
+After you learn about the hub cluster and managed cluster architecture, learn about the link:../manage_cluster#supported-clouds[Supported clouds], which lists the cloud provider cluster options.
 
 The hub cluster is a Red Hat OpenShift cluster version 4.3.x or 4.4 and can run on any https://docs.openshift.com/container-platform/4.3/architecture/architecture-installation.html[supported Red Hat OpenShift Container Platform infrastructure].
 
 The xref:../about/glossary_terms.adoc#glossary-of-terms[Glossary of terms] defines common terms for the product.
 
+If you experience problems, see the link:troubleshoot_acm/troubleshooting.adoc[Troubleshooting guide] to learn about the `mustgather` command and see documented troubleshooting tasks that might help resolve issues.
+
 [#install]
```
Comment (swopebe) [inline about/quick_start.adoc:17]: link to t-shooting here as this QSG will be shared.

**Review by dockerymick — COMMENTED** [review]
Updated links thanks for the updates!

**Pull request merged**